test/powershell/Host/ScreenReader.Tests.ps1

```diff
@@ -44,8 +44,12 @@ Describe "Validate start of console host" -Tag CI {
     }
 
     It "PSReadLine should not be auto-loaded when screen reader status is active" -Skip:(-not $IsWindows) {
+        if ([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture -eq [System.Runtime.InteropServices.Architecture]::Arm64) {
+            Set-ItResult -Pending -Because "TBD, PSReadline loads on ARM64"
```

Comment:
Change TBD to "investgate"

Reply:
I'd prefer we kept capitalisation & correct spelling so should be `Investigate`